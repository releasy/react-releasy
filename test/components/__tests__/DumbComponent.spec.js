import * as React from 'react';
import {  shallow } from 'enzyme';

import DumbComponent from '../DumbComponent';

it('should render a empty DumbComponent', () => {
  const component = shallow(<DumbComponent />);

  expect(component.instance()).toMatchSnapshot();
});
