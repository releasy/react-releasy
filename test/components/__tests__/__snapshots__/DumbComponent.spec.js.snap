// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a empty DumbComponent 1`] = `null`;
